### PR TITLE
KubeSphere Meetup 2020

### PR DESCRIPTION
**Pull request by JiannaLiu**

### CODE REVIEW

File: content/en/news/kubesphere-meetup-2020.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+![Snip20210201_23](/images/news/meetup-2020/Snip20210201_23.png)
+
+![1.pic](/images/news/meetup-2020/1.pic.jpg) 
+
+More importantly, we look to further renew and strengthen our bonds and values in the community. And this is what has been driving KubeSphere for so long. Community members contribute, cooperate in and come up with innovative ideas, which is exactly what open source is all about.
+
+ ![Snip20210201_26](/images/news/meetup-2020/Snip20210201_26.png)
+
+The meetup was a great success and it was not possible without community members. Thank you all for your support and see your next year in the meetup.
```
Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:63]: see your next year
->
see you next year

File: content/en/news/kubesphere-meetup-2020.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+We also provided on-site workshops to discuss and try the most common use features of KubeSphere.
+
+ ![Snip20210201_20](/images/news/meetup-2020/Snip20210201_20.png)
+
+![Snip20210201_22](/images/news/meetup-2020/Snip20210201_22.png)
+
+ 
+
+KubeSphere has been recognized by many uses across the world and has developed a more diversified and vibrant community. Though so many passionate users could not present at the event in person, they generously shared their experience and expectations for KubeSphere in videos.
```
Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:45]: by many uses
->
by many users

Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:45]: a more diversified and vibrant community
->
a diversified and vibrant community

Delete "more" here as we do not have any comparison here.

File: content/en/news/kubesphere-meetup-2020.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+keywords: KubeSphere, Community, Meetup, Kubernetes
+description: KubeSphere and Friends 2020 Meetup attracted more than 100 individual community members and honored guests to share their ideas and experiences on cloud-native and Kubernetes.
+createTime: '2021-02-02'
+author: 'Feynman, Sherlock'
+
+---
+
+![Snip20210201_8](/images/news/meetup-2020/Snip20210201_8.png)
+
+Around the latest technology trend – 'cloud-native,' the annual event KubeSphere and Friends 2020, also known as Kubernetes and Cloud-Native Meetup, was held on December 19th, 2020 in Beijing, attracting more than one hundred Kubernetes enthusiasts and KubeSphere community members.
```
Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:13]: Around the latest technology trend – 'cloud-native,'
->
Around the latest technology trend – 'cloud-native',

The single quotation mark can be deleted here, I reckon.

File: content/en/news/kubesphere-meetup-2020.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+
+title: 'Recap: KubeSphere & Friends 2020 Meetup'
+keywords: KubeSphere, Community, Meetup, Kubernetes
+description: KubeSphere and Friends 2020 Meetup attracted more than 100 individual community members and honored guests to share their ideas and experiences on cloud-native and Kubernetes.
+createTime: '2021-02-02'
+author: 'Feynman, Sherlock'
```
Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:7]: Add your name here.

File: content/en/news/kubesphere-meetup-2020.md (hunk 3 above)
Comment (Sherlock113) [inline content/en/news/kubesphere-meetup-2020.md:13]: one hundred
->
100

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments.

File: content/en/news/kubesphere-meetup-2020.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+![Snip20210201_8](/images/news/meetup-2020/Snip20210201_8.png)
+
+Around the latest technology trend – 'cloud-native,' the annual event KubeSphere and Friends 2020, also known as Kubernetes and Cloud-Native Meetup, was held on December 19th, 2020 in Beijing, attracting more than one hundred Kubernetes enthusiasts and KubeSphere community members.
+
+Not only did our individual community members participate in the event, our guests from CNCF and AWS, and other enterprise users all shared their ideas and experiences from different perspectives.
```
Comment (FeynmanZhou) [inline content/en/news/kubesphere-meetup-2020.md:15]: Not only did our individual community members participate in the event =>
Not only did developers from the KubeSphere team participate in the event

File: content/en/news/kubesphere-meetup-2020.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+
+
+![Snip20210201_10](/images/news/meetup-2020/Snip20210201_10.png)
+
+​                              *Ray Zhou   Head of KubeSphere*
+
+
+
+We also provided on-site workshops to discuss and try the most common use features of KubeSphere.
```
Comment (FeynmanZhou) [inline content/en/news/kubesphere-meetup-2020.md:37]: workshops => workshop

**Pull request merged**